pkg/transfers/inbound/cleanup.go

```diff
@@ -36,6 +36,10 @@ func Cleanup(logger log.Logger, agent upload.Agent, dl *downloadedFiles) error {
 
 // DeleteZeroByteFiles deletes empty ACH files if file older than value in config
 func DeleteZeroByteFiles(logger log.Logger, agent upload.Agent, dl *downloadedFiles, after time.Duration) error {
+	if after <= 0 * time.Second {
+		logger.Log(fmt.Sprintf("deleting empty file requires after > 0 %s", after))
```

Comment:
I think this needs `%v` for `time.Duration` values instead of `%s`.

Reply:
Oh, CI is saying this file isn't formatted either.  I bet it's the `gofmt` requirements around spaces in math equations

```
./pkg/transfers/inbound/cleanup.go is not formatted
```